src/Link.js

```diff
@@ -24,24 +24,26 @@ const hoverColor = system({
   }
 })
 
-const Link = styled.a`
+const Link = styled.a.attrs(props => ({
+  color: props.color ? props.color : props.muted ? 'gray.6' : 'blue.5'
+}))`
   text-decoration: ${props => (props.underline ? 'underline' : 'none')};
   &:hover {
-    text-decoration: underline;
-    ${hoverColor};
+    text-decoration: ${props => (props.muted ? 'none' : 'underline')};
+    ${props => (props.hoverColor ? hoverColor : props.muted ? `color: ${get('colors.blue.5')(theme)}` : '')};
```

Comment:
You'll want to make the `hoverColor` prop theme compatible as well:
```suggestion
    color: ${props => (props.hoverColor ? get(hoverColor)(theme) : props.muted ? get('colors.blue.5')(theme)` : 'inherit'};
```